Networking/Networking/Mapper/ShippingLabelPrintDataMapper.swift

```diff
@@ -0,0 +1,25 @@
+import Foundation
+
+
+/// Mapper: Shipping Label Print Data
+///
+struct ShippingLabelPrintDataMapper: Mapper {
```

Comment:
nit: what do you think about adding the unit tests for this mapper as we did for the other `*Mapper` struct?

Reply:
I thought about adding unit tests for `ShippingLabelPrintDataMapper`, but then they are going to be the same as in `ShippingLabelRemoteTests` on the remote. Since `ShippingLabelRemote.printShippingLabel` uses the mapper for its result, the mapper is tested in `ShippingLabelRemoteTests.test_printShippingLabel_returns_ShippingLabelPrintData`. Lemme know if you have other test scenarios in mind for the mapper and I can add them separately!

Reply:
Sounds good to have just the the `ShippingLabelRemoteTests` here since they are the same 👍